source/core/ScriptDomain.cs

```diff
@@ -35,6 +35,54 @@ public class ScriptDomain : MarshalByRefObject, IDisposable
 		bool[] keyboardState = new bool[256];
 		List<Assembly> scriptApis = new List<Assembly>();
 
+		#region Instance Fields And Methods for Thread Local Storage
+
+		unsafe delegate* unmanaged[Stdcall]<IntPtr> GetTls;
```

Comment:
These are not using the stcall calling convention as per their declaration in DllMain.cpp, but cdecl. Though technically the two don't even exist in x64 and are replaced by the x64 calling convention by the compiler, so that's why it would still work, just a code correctness thing. As such it's probably best to just use the platform default by not specifying one at all: `delegate* unmanaged<IntPtr>`.

Reply:
Looks like you _have_ to specify some calling convention in .NET Framework projects because otherwise the compiler gives  a CS8889 error.